ott/problems/linear/potentials.py

```diff
@@ -167,15 +169,6 @@ def __init__(
       a: Optional[jnp.ndarray] = None,
```

Comment:
This constructor will be mostly used by the `SinkhornOutput` method `to_dual_potentials`, but should the user ever use it by herself, not passing `a` and `b` specifically could be a problem. Therefore I think I don't think `a` and `b` should be marked as optional.

Reply:
Done.